portfolio/src/main/webapp/style.css

```diff
@@ -41,6 +41,36 @@ h3 {
   margin-bottom: 5px;
 }
 
+/* Add a black background color to the top navigation */
+.topnav {
+  background-color: rgb(0, 0, 0);
+  overflow: hidden;
+  border-radius: 10px;
+  margin: 10px;
+}
+
+/* Style the links inside the navigation bar */
+.topnav a {
+  float: left;
+  color: #f2f2f2;
+  text-align: center;
+  padding: 10px 10px;
+  text-decoration: none;
+  font-size: 17px;
+}
+
+/* Change the color of links on hover */
+.topnav a:hover {
+  background-color: #ddd;
+  color: black;
+}
+
+/* Add a color to the active/current link */
+.topnav a.active {
```

Comment:
bonus: you can use the [`:active` selector](https://www.w3schools.com/cssref/sel_active.asp) here instead of having a CSS class